server/ngwmn/templates/macros/components.html

```diff
@@ -34,59 +35,107 @@ <h3 class="usa-alert-heading">Warning</h3>
+                                <th scope="col"></th>
+                                <th scope="col">Depth</th>
+                                <th scope="col">Description</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {% for item in well_log.construction %}
+                                <tr class="{{ item.type }}" data-local-id="{{ item.id }}">
+                                    <td>
+                                        {% if item.type == 'screen' %}
+                                            <svg width="30" height="30" viewBox="0 0 30 30">
+                                                <rect width="30" height="30"
+                                                      fill="url(#screen-pattern-{{ loop.index0 % 2 }})"/></svg>
+                                            </svg>
+                                        {% else %}
+                                            <svg width="30" height="30" viewBox="0 0 30 30">
+                                                <line x1="15" y1="0" x2="15" y2="30" stroke-width="2" stroke="gray"></line>
+                                            </svg>
+                                        {% endif %}
+                                    </td>
+                                    <td>{{ item.position.coordinates.start }}-{{ item.position.coordinates.end }} {{ item.position.unit }}</td>
+                                    <td>
+                                        {% if item.diameter and item.diameter.value %}
+                                            {{ item.diameter.value }} {{ item.diameter.unit }} diameter
+                                        {% endif %}
+                                        {% if item.material != 'Screen, Type Not Known' %}
+                                            {{ item.material | lower() }}
+                                        {% endif %}
+                                        {{ item.type | lower() }}
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                            </tbody>
+                        </table>
+                    {% endif %}
+                </div>
+
+                <input name="tabs" type="radio" id="tab-2" class="inputRadioButtonTab"/>
+                <label for="tab-2" class="labelRadioButtonTab">Detailed Lithology</label>
+                <div class="panel">
+                    {% if well_log.log_entries is not defined %}
+                        <p>No lithology data is available</p>
+                    {% else %}
+
+                        <table class="usa-table usa-table-borderless lithologyTable">
+                            <thead>
+                                <tr>
+                                    <th scope="col">Depth from</th>
+                                    <th scope="col">Depth to</th>
```

Comment:
On the construction charts, to save space I combined depth from/to into a single column. What would you think about doing that here?

```suggestion
                                     <th scope="col">Depth</th>
```

Reply:
Seems like a good idea. Done.